src/index.js

```diff
@@ -1,3 +1,3 @@
 'use strict';
 
-module.exports = require('@oclif/command');
+module.exports = require('@oclif/core');
```

Comment:
I see that we have `LICENSE-THIRD-PARTY.txt` which still mentions `@oclif/command`. Should we update it too?